tests/test_file.py

```diff
@@ -15,23 +15,27 @@
 from v3iofs import V3ioFile, V3ioFS
 
 
-def test_fetch_range(fs: V3ioFS, tmp_obj):
-    v3f = V3ioFile(fs, tmp_obj.path)
-    start, end = 3, len(tmp_obj.data) - 3
+def test_fetch_range(fs: V3ioFS, tmp_file):
+    v3f = V3ioFile(fs, tmp_file.path)
+    start, end = 3, len(tmp_file.data) - 3
     data = v3f._fetch_range(start, end)
-    expected = tmp_obj.data[start:end]
+    expected = tmp_file.data[start:end]
+    print("expected...")
```

Comment:
These prints are not needed. pytest will show the diff if there's an error.

Reply:
Removed per request.